components/src/instrument/InstrumentDiagram.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
 import { FlattenSimpleInterpolation } from 'styled-components'
-import { Box } from '..'
+import { Flex, ALIGN_CENTER, JUSTIFY_CENTER } from '..'
```

Comment:
nit: lets specify the exact import location for all of these so we don't run into circular dependency issues down the road.

```suggestion
import { Flex } from '../primitives'
import { ALIGN_CENTER, JUSTIFY_CENTER } from '../styles'
```